ui/src/app/views/table.html

```diff
@@ -1,12 +1,14 @@
 <div class="table-responsive-vertical md-whiteframe-z1">
-    <table id="table" class="table table-hover table-bordered">
+    <md-button class="md-primary md-raised" layout-align="center end" ng-click="changeView()"><i class="material-icons ng-scope" id="grid-button">view_module</i></md-button>
```

Comment:
`view_module`?

Reply:
The content of the `<i>` should be `view_module` for it to render?

Reply:
I also don't understand `view_module`

Reply:
Ok. Now I get it. 
WDYT about the following approach.
here you modify it as
`.......id="grid-button">{{ isGridVisible ?  'view_module' : 'view_list' }}</i>` . Then you can eliminate the `document.getElement.......` part.

Reply:
Done 👍